components/Carousel/slick.js

```diff
@@ -0,0 +1,265 @@
+    opacity: 1;
+  }
+  .slick-prev.slick-disabled,
+  .slick-next.slick-disabled {
+    opacity: 0.25;
+  }
+
+  .slick-prev:before,
+  .slick-next:before {
+    font-size: 20px;
+    line-height: 1;
+    opacity: 0.75;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
+  .slick-prev {
+    left: -25px;
+  }
+  [dir='rtl'] .slick-prev {
+    right: -25px;
+    left: auto;
+  }
+  .slick-prev:before {
+    content: '←';
+  }
+  [dir='rtl'] .slick-prev:before {
+    content: '→';
+  }
+
+  .slick-next {
+    right: -25px;
+  }
+  [dir='rtl'] .slick-next {
+    right: auto;
+    left: -25px;
+  }
+  .slick-next:before {
+    content: '→';
+  }
+  [dir='rtl'] .slick-next:before {
+    content: '←';
+  }
+
+  /* Dots */
+
+  .slick-dotted.slick-slider {
+    margin-bottom: 30px;
+  }
+
+  .slick-dots {
+    position: absolute;
+    bottom: -25px;
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-align: center;
+  }
+
+  .slick-dots li {
+    position: relative;
+    \display: inline-block;
+    width: 20px;
+    height: 20px;
+    margin: 0 5px;
+    padding: 0;
+    cursor: pointer;
+  }
+
+  .slick-dots li button {
+    font-size: 0;
+    line-height: 0;
+    display: block;
+    width: 20px;
+    height: 20px;
+    padding: 5px;
+    cursor: pointer;
+    border: 0;
+    outline: none;
+    background: transparent;
+  }
+
+  .slick-dots li button:hover,
+  .slick-dots li button:focus {
+    outline: none;
+  }
+  .slick-dots li button:hover:before,
+  .slick-dots li button:focus:before {
+    opacity: 1;
+  }
+  .slick-dots li button:before {
+    font-size: 8px;
+    line-height: 20px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 20px;
+    height: 20px;
+    content: '•';
+    text-align: center;
+    opacity: 0.25;
+    ${({
+      theme: {
+        colors: {
+          secondary: { 700: secondary700 },
+        },
+      },
+    }) => `
+      color: ${secondary700};
+    `};
+
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
+  .slick-dots li.slick-active button:before {
+    opacity: 0.75;
+    ${({
+      theme: {
+        colors: {
+          secondary: { 700: secondary700 },
+        },
+      },
+    }) => `
+      color: ${secondary700};
+    `};
+  }
+
+  .slick-slider {
+    position: relative;
+    display: block;
+    box-sizing: border-box;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-touch-callout: none;
+    -khtml-user-select: none;
+    -ms-touch-action: pan-y;
+    touch-action: pan-y;
+    -webkit-tap-highlight-color: transparent;
+  }
+
+  .slick-list {
+    position: relative;
+    display: block;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+  }
+  .slick-list:focus {
+    outline: none;
+  }
+  .slick-list.dragging {
+    cursor: pointer;
+    cursor: hand;
```

Comment:
`cursor: hand` should be only in mobile/tablet version?

Reply:
Testing here, this option is not active in actual cenario in quantum. We decided to keep the css file from the lib in this way for future changes. Is possible that have future changes and this settins be used.